openmls/src/group/mls_group/mod.rs

```diff
@@ -123,21 +123,21 @@ pub enum MlsGroupState {
     Inactive,
 }
 
-/// A `MlsGroup` represents an CoreGroup with
-/// an easier, high-level API designed to be used in production. The API exposes
+/// A `MlsGroup` represents an MLS group with
+/// an easy, high-level API designed to be used in production. The API exposes
```

Comment:
```suggestion
/// a high-level API. The API exposes
```
Since we don't expose `CoreGroup` anymore, there's no need to be specific about where this is meant to be used.